.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -52,7 +52,9 @@ module.exports.run = async function run( { github, context } ) {
 	for ( const currentComment of comments ) {
 		if (
 			currentComment.user.type === 'Bot' &&
-			currentComment.body.includes( 'Test using WordPress Playground' )
+			currentComment.body.includes(
+				'Test this PR in the WordPress Playground'
```

Comment:
To match the new shorter message here: https://github.com/WordPress/gutenberg/pull/62393/commits/d2cc022bd5caf930d19158cbe4d7cf7518b75a2a.